Sources/App/Views/Utils.swift

```diff
@@ -3,11 +3,11 @@ import Plot
 
 // MARK: - Pluralisation
 
-func pluralizedCount(_ count: Int, singular: String, plural: String? = nil) -> String {
+func pluralizedCount(_ count: Int, singular: String, plural: String? = nil, capitalized: Bool = false) -> String {
```

Comment:
I'd suggest instead of an additional parameter we let the caller do

```
pluralizedCount(products.libraries, singular: "library", plural: "libraries").capitalized
```

using `String.capitalized`.

Reply:
It ignores numbers, so it's the same thing.